.travis.yml

```diff
@@ -17,13 +17,6 @@ cache:
 before_install: docker build -t axe-selenium-python .
 install: skip
 script:
-  - docker run -e TOXENV=$TOXENV axe-selenium-python tox
+    - docker run -e TOXENV=$TOXENV axe-selenium-python tox
```

Comment:
I may be wrong, it looks like the indentation is off here?

Reply:
Yes...the indentation on this file keeps changing, and I'm not exactly sure why/when this is occurring.